### PR TITLE
Add check for record['discount'] = None

### PR DESCRIPTION
**Pull request by mdietrichc2c**

### CODE REVIEW

**Comment by mdietrichc2c** [comment]
Quick PR for urgent issue: the record can be {'discount_amount': None}, and float() fails.
This just adds a check.

File: magentoerpconnect/sale.py [hunk 1]
```diff
@@ -1030,10 +1030,12 @@ class SaleOrderLineImportMapper(ImportMapper):
...
+        if record.get('discount_amount'):
+            discount_value = float(record.get('discount_amount', 0))
+        if self.backend_record.catalog_price_tax_included and \
+           record.get('row_total_incl_tax'):
             row_total = float(record.get('row_total_incl_tax', 0))
-        else:
+        elif record.get('row_total'):
             row_total = float(record.get('row_total', 0))
         discount = 0
         if discount_value > 0 and row_total > 0:
```
Comment (rdeheele) [inline magentoerpconnect/sale.py:1039]: @mdietrichc2c with your correction, if discount_amount is not in record dict, discount_value variable will not be assigned

**Comment by mdietrichc2c** [comment]
@rdeheele yep, sorry; I spotted it after the inital commit. They're initialized now.

**Comment by mdietrichc2c** [comment]
It's now consistent with https://github.com/OCA/connector-magento/pull/70

**Comment by guewen** [comment]
:+1:

**Comment by lepistone** [comment]
OCA#70 is now merged. Can we close this one?

**Comment by mdietrichc2c** [comment]
@lepistone not really, that one is for the pending merge. Can you merge it instead?

**Comment by lepistone** [comment]
Not sure, I'll ping @guewen instead.

**Pull request merged**